lib/Dialect/FIRRTL/Transforms/InferResets.cpp

```diff
@@ -741,7 +741,8 @@ void InferResetsPass::traceResets(CircuitOp circuit) {
         .Case<RefSendOp>([&](auto op) {
           // Trace using base types.
           traceResets(op.getType().getType(), op.getResult(), 0,
-                      op.getBase().getType(), op.getBase(), 0, op.getLoc());
+                      op.getBase().getType().getPassiveType(), op.getBase(), 0,
```

Comment:
I think this is only important, presently, for debug logs and when reporting errors.